README.md

```diff
@@ -31,6 +31,13 @@ Bazel team, provides support for the code. However, this repository is part of
 the test suite used to vet new Bazel releases. See the [How to
 contribute](CONTRIBUTING.md) page for information on our development workflow.
 
+## `bzlmod` support
+
+- Status: Beta
+- Full Feature Parity: No
+
+See [here](BZLMOD_SUPPORT.md) for more details.
```

Comment:
nit: instead of linking "this" or "here", use a descriptive document title
```suggestion
See [Bzlmod support](BZLMOD_SUPPORT.md) for more details.
```